web/client/components/import/style/StylePanel.jsx

```diff
@@ -239,6 +258,31 @@ class StylePanel extends React.Component {
             </Col>
         </>
     );
+
+    checkAndDisableStyleCustomization = () => {
+        this.setState({disableStyleCustomization: false, useDefaultStyle: false});
+        const layer =  this.props.selected;
+        if (layer) {
+            if (layer.features.length) {
```

Comment:
you can remove one if in this case
```suggestion
        if (layer && layer.features.length) {
```